### DISCUSSION TITLE
Superadmin member list

### DISCUSSION THREAD

**Discussion by Feringard** [snipe/snipe-it#12741] [Q&A]
Hi all,

I would like to ask for help, with pulling a report on all of the superadmin in my account.
I tried to look thru the people tab, played around with filters and I still can't figure this one out. 

Any help or hints will be much appreciated

Regards
Marcin

**Comment by snipe** [comment]
Hi there - are you using group level permissions or user-level permissions, or some combo of both?

**Reply by Feringard** [reply]
Hey hey, a combo, but mostly user level.

**Reply by snipe** [reply]
Hm, okay - do you have access to a mysql command line (or GUI)? There isn't an easy way to do this via the GUI, but I can give you a few queries that will help you get what you need. 

```sql
SELECT username FROM users  WHERE permissions LIKE '%"superuser":"1"%';
```

to get the users with superadmin permissions:

```sql
SELECT users.username, permission_groups.permissions FROM users, permission_groups, users_groups WHERE permission_groups.permissions LIKE '%"superuser":"1"%' AND users_groups.user_id=users.id;
```

**Reply by Feringard** [reply]
Thank you for the response. Where may I get in to that mysql command line/gui? I am not a sql expert :)
